app/src/app/content-plugin/components/header/header.component.ts

```diff
@@ -42,11 +42,18 @@ import { isContentServiceEnabled } from '@alfresco/aca-shared/rules';
 })
 export class AppHeaderComponent implements OnInit, OnDestroy {
   private onDestroy$: Subject<boolean> = new Subject<boolean>();
+
   @Output()
   toggleClicked = new EventEmitter();
 
   @Input() expandedSidenav = true;
 
+  @Input() data: { layout?: SidenavLayoutComponent; isMenuMinimized?: boolean } = {};
+
+  get isSidenavExpanded(): boolean {
+    return !this.data.isMenuMinimized ?? this.expandedSidenav;
```

Comment:
why double flags here, you can probably remove the `expandedSidenav` if you take this value from the `data`?